<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSession</class>
 <widget class="QDialog" name="NewSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建会话</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="sesType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>串口终端</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>telnet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Console</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>网络助手</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kconfig</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>14</y>
     <width>38</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>名称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sesName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>会话1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="fmParam">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>参数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewSession</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewSession</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
